src/cli/cli.rs

```diff
@@ -270,20 +270,16 @@ pub fn build_cli() -> App<'static, 'static> {
         .subcommand(
             SubCommand::with_name("gradient")
                 .about("Generate an interpolating sequence of colors")
-                .long_about("Generate a sequence of colors that interpolates between 'start' and \
-                            'stop'. The interpolation is performed in the specified color space.\n\n\
+                .long_about("Generate a sequence of colors that interpolates between the specified colors.\n\
+                            The interpolation is performed in the specified color space.\n\n\
                             Example:\n  \
-                              pastel gradient --colorspace=HSL ffffcc fd8d3c")
+                              pastel gradient --colorspace=HSL ffffcc fd8d3c\n  \
+                              pastel gradient --colorspace=HSL ab47bc f06292 f44336")
```

Comment:
Maybe add a more "exciting" example here (where the middle color has a more significant role), like a "cold" to "hot" transition:
```
pastel gradient 555ee4 white d84341 -n 15
```